src/components/dashboard/data/dashboardLoader.ts

```diff
@@ -31,8 +32,27 @@ const makeDashboardLoader: Types.MakeRouteLoaderFunctionWithQueryClient = functi
       authenticatedUser,
       enterpriseSlug,
     });
-    await Promise.all([
+
+    const loadEnrollmentsPoliciesAndRedirectForNewUsers = Promise.all([
       queryClient.ensureQueryData(queryEnterpriseCourseEnrollments(enterpriseCustomer.uuid)),
```

Comment:
[inform] Once the Dashboard BFF endpoint is integrated into the `dashboardLoader`, it will replace this `queryEnterpriseCourseEnrollments`.